polling_stations/apps/data_finder/helpers/geocoders.py

```diff
@@ -39,7 +34,7 @@ def geocode(self):
 
 
 class OnspdGeocoderAdapter(BaseGeocoder):
-    def geocode(self):
+    def geocode(self, uprn=None):
```

Comment:
Why can we pass `OnspdGeocoderAdapter.geocode()` a UPRN that isn't used anywhere in the function?